README.md

```diff
@@ -26,4 +26,4 @@ For more info about the NL Design System and learn about things happening in our
 
 This project is free and open-source software licensed under the [European Union Public License (EUPL) v1.2](LICENSE.md). Documentation is licensed as [Creative Commons Zero 1.0 Universal (`CC0-1.0`)](https://creativecommons.org/publicdomain/zero/1.0/legalcode). Applying design elements from this project is strictly prohibited for organisations that are not part of the City of Amsterdam.
 
-For information about proprietary assets in this repository, please carefully read the [NOTICE file](NOTICE.md).
+The assets in the proprietary folder are not covered by the open source licenses. Each folder in the proprietary folder specifies the license that applies to that folder. For more information about proprietary assets, please carefully read the [NOTICE file](NOTICE.md).
```

Comment:
We could enclose directory names in backticks, e.g. `proprietary`, and I think we should (talking about conformance keywords…) replace the term ‘folder’ with ‘directory’.